app/controllers/journeys/tradeDetails/CheckYourSelfEmploymentDetailsController.scala

```diff
@@ -41,7 +41,7 @@ class CheckYourSelfEmploymentDetailsController @Inject()(override val messagesAp
                                                          view: CheckYourSelfEmploymentDetailsView)
                                                         (implicit val ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
 
-  def onPageLoad(taxYear: Int, businessId: String): Action[AnyContent] = (identify andThen getData) async { //TODO (Edem) does this need 'andThen requireData' ?
+  def onPageLoad(taxYear: Int, businessId: String): Action[AnyContent] = (identify andThen getData) async { //TODO Business does this need 'andThen requireData' ?
```

Comment:
Do we have an answer for this?

Reply:
Yes, spoke with Chris. It likely will need a require data but this whole section of the service is going to change and the journey isn't built yet so will be left without till further notice.